package.json

```diff
@@ -15,9 +15,9 @@
     "test:ci:serve": "cross-env VITE_PRESERVE_BUILD_ARTIFACTS=1 jest --verbose --no-cache --runInBand --force-exit --ci --json --outputFile=\"temp/serve/jest-results.json\"",
     "test:ci:build": "cross-env VITE_TEST_BUILD=1 VITE_PRESERVE_BUILD_ARTIFACTS=1 jest --verbose --no-cache --runInBand --force-exit --ci --json --outputFile=\"temp/build/jest-results.json\"",
     "lint": "eslint --ignore-path .gitignore '**/*.{js,ts,svelte,html,svx,md}'",
-    "lint:fix": "pnpm lint -- --fix",
+    "lint:fix": "pnpm run lint -- --fix",
```

Comment:
I think pnpm added a feature recently to allow you to drop the `run` part. Either way I feel like this bloats the PR - makes it harder to review.

Reply:
that feature made me add it back, i wasn't sure if it would prefer a `lint` dependency  over the lint script.